docs/api.md

```diff
@@ -1440,6 +1441,77 @@ Failing requests may return the following error:
 
 * status code 400, errno 102: unknown account
 
+## POST /v1/account/devices/notify
+
+:lock: HAWK-authenticated with the sessionToken.
+
+Notifies a set of devices in the caller's account of an event
+by sending a Push notification.
+
+### Request
+
+___Headers___
+
+The request must include a Hawk header that authenticates the request
+using a `sessionToken` received from `/v1/account/create` or `/v1/account/login`.
```

Comment:
Please also add a `___Parameters___` section to document the request body paraemeters.